server/sonar-server/src/main/java/org/sonar/server/qualitygate/ws/UnsetDefaultAction.java

```diff
@@ -19,18 +19,16 @@
  */
 package org.sonar.server.qualitygate.ws;
 
+import org.sonar.api.server.ws.Change;
 import org.sonar.api.server.ws.Request;
 import org.sonar.api.server.ws.Response;
 import org.sonar.api.server.ws.WebService;
-import org.sonar.server.qualitygate.QualityGates;
+import org.sonar.server.ws.RemovedWebServiceHandler;
 import org.sonarqube.ws.client.qualitygate.QualityGatesWsParameters;
 
 public class UnsetDefaultAction implements QualityGatesWsAction {
 
-  private final QualityGates qualityGates;
-
-  public UnsetDefaultAction(QualityGates qualityGates) {
-    this.qualityGates = qualityGates;
+  public UnsetDefaultAction() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1186)